src/config/bot.ts

```diff
@@ -59,7 +61,11 @@ export async function startBot(database: Database) {
 	const bot = new TelegramBot<CustomContext>(process.env.TOKEN)
 	extendContext(bot, database)
 	setupMiddlewares(bot, i18n)
+	// NOTE(DP): I guess we need to log users actions, e.g. start, stop, etc
```

Comment:
Does not look useful / will be a lot of entries

Reply:
@ExposedCat Ideally, general recommendation to treat logs as events… there is a good article w/ explorations, but I cannot find it quickly :(

anyway, in this case we don't create extra logs only for useful for debugging things